pkg/authorization/service.go

```diff
@@ -1310,3 +1410,69 @@ func (a *AuthService) getDomainTypeFromDomain(domain string) string {
 	}
 	return ""
 }
+
+// setupDefaultOrganizationRoleMetadata creates metadata for default organization roles
+func (a *AuthService) setupDefaultOrganizationRoleMetadata(orgID string) error {
+	defaultRoles := []struct {
+		name        string
+		displayName string
+		description string
+	}{
+		{
+			name:        RoleOrgOwner,
+			displayName: "Owner",
+			description: "Full control over organization settings, billing, and member management.",
+		},
+		{
+			name:        RoleOrgAdmin,
+			displayName: "Admin",
+			description: "Can manage canvases, users, groups, and roles within the organization.",
+		},
+		{
+			name:        RoleOrgViewer,
+			displayName: "Viewer",
+			description: "Read-only access to organization resources and information.",
+		},
+	}
+
+	for _, role := range defaultRoles {
+		if err := models.UpsertRoleMetadata(role.name, DomainOrg, orgID, role.displayName, role.description); err != nil {
+			return fmt.Errorf("failed to upsert role metadata for %s: %w", role.name, err)
+		}
+	}
+
+	return nil
+}
+
+// setupDefaultCanvasRoleMetadata creates metadata for default canvas roles
+func (a *AuthService) setupDefaultCanvasRoleMetadata(canvasID string) error {
+	defaultRoles := []struct {
+		name        string
+		displayName string
+		description string
+	}{
+		{
+			name:        RoleCanvasOwner,
+			displayName: "Owner",
```

Comment:
We should not use hardcoded values. "Owner" is hardcoded here and in `pkg/models/role_metadata.go`. We should move all these values into the `pkg/models` package.